### ISSUE TITLE
what is the purpose of geoip module in PHP and NGINX images?

### ISSUE THREAD

**Issue by virtuman** [wodby/php#59]
It looks like geoip is being installed and activated, but it isn't really usable until the database files are loaded and proper nginx headers are passed to proxy or fastcgi backend. 

Is there a way that this was intended to be used or 

**Example for geoip:**
_Download database:_
https://github.com/lalamove/docker-nginx-php7/blob/master/Dockerfile#L190
_Include in nginx.conf:_
https://github.com/lalamove/docker-nginx-php7/blob/master/nginx_log_format.conf#L2
_Append fastcgi or proxy headers:_
https://github.com/lalamove/docker-nginx-php7/blob/master/nginx_geoip_params

**Example for geoip2:**
_Download databases:_
https://github.com/crazy-max/docker-matomo/blob/master/Dockerfile#L77
_Include in nginx.conf and define parameters:_
https://github.com/crazy-max/docker-matomo/blob/master/assets/tpls/etc/nginx/nginx.conf#L9
_Append fastcgi headers:_
https://github.com/crazy-max/docker-matomo/blob/master/assets/tpls/etc/nginx/nginx.conf#L123

**PHP geoip support**
Same goes for php - since it doesn't define location of maxMind's database, it will generate php warning: like this one:
`Warning: geoip_country_code_by_name(): Required database not available at /GeoIP.dat. in /var/www/html/geo.php on line 1`

**Comment by csandanov** [comment]
Thank you for reporting this. IIRC geoip extension was originally added to meet requirements for Matomo (Piwik) during installation. Since 3.5.0 Matomo [supports geoip2](https://matomo.org/docs/geo-locate/), so I guess they no longer require geoip extension (that's something I need to check) and use some composer package because the geoip extension AFAIK does not support geoip2. In addition, since Jan 2, 2019 maxmind dropped support for [legacy databases](https://support.maxmind.com/geolite-legacy-discontinuation-notice/), so you can't actually download now `GeoIP.dat` from the orignial source. With that being said, we should probably just remove geoip extension and expect that user will download a geo location composer library and the database file during a build of a runtime container. 

As for Nginx, I don't think there's a point in passing through fastcgi params considering what I described above. The only value I see in geo locating before PHP is for caching but for this exact reason we've recently revamped [wodby/varnish](https://github.com/wodby/varnish) image so it now has geo ip module with the database (currently the legacy version), it passes geo headers in `X-` headers.

**Comment by csandanov** [comment]
Ok, so matomo actually does not require geoip extension and supports geoip2, all it needs is to download maxmind geolite databases. I think it's a preferred way to do so since the functionality of composer maxmind library will always be wider than nginx module and database will constantly update during runtime builds. I don't think there's a point of having geoip modules on application's Nginx, maybe it makes sense to have it on a reverse proxy for some reasons, IDK. For caching purposes we already have geoip with legacy db on varnish.

So, to recap: we've removed geoip modules from both php and nginx images. Having it on nginx makes no sense and for PHP you should download geoip2 lite dbs during builds (legacy `.dat` dbs no longer supported) + use appropriate composer library.
